src/mqm/geo_process.py

```diff
@@ -209,16 +210,30 @@ def bounding_box_process(self):
         end_point = 0
 
         
-        # loop through all geojson files
-        for f in os.listdir(self.folder_path):
-            # load the Geo-json file and ignore other files
-            if os.path.splitext(os.path.join(self.folder_path, f))[1] == '.geojson':
+        # Loop through all geojson files in folder path recursively
+        for roots, dirs, files in os.walk(self.folder_path):
+            checks_list = []
+            for f in files:
+                check_name = os.path.splitext(f)[0].split('-')[0]
+                if not check_name.startswith('.'):
+                    checks_list.append(check_name)
+            if len(checks_list) != len(set(checks_list)):
+                sys.exit('Each subdirectory must have only one GeoJSON per Atlas Check and no more than one boundary '
```

Comment:
This was originally implemented to prevent misplacement of geojsons from multiple country sets into the same directory, but based on this information, it makes more sense to remove this condition. I've taken this line out in the latest commit.